From 814035bd253945de9311b3defede8f17ccae8715 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Tue, 19 Nov 2019 13:48:52 +0100
Subject: [PATCH 2/2] Remove undefines

Fastboot support removes the definitions
added by the default mx8m board config file.

Remove the changes that were done for fastboot
support.

Upstream-status: Inappropriate[configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 include/configs/imx8mq_evk_android.h       | 8 --------
 include/configs/imx8mq_evk_androidthings.h | 7 -------
 include/configs/imx8qm_arm2_android.h      | 8 --------
 include/configs/imx8qxp_arm2_android.h     | 8 --------
 include/configs/imx8qxp_mek_android.h      | 7 -------
 5 files changed, 38 deletions(-)

diff --git a/include/configs/imx8mq_evk_android.h b/include/configs/imx8mq_evk_android.h
index ea1902940d..48b73a2f5e 100644
--- a/include/configs/imx8mq_evk_android.h
+++ b/include/configs/imx8mq_evk_android.h
@@ -43,12 +43,4 @@
 #define CONFIG_SUPPORT_RAW_INITRD
 #define CONFIG_SERIAL_TAG
 
-#undef CONFIG_EXTRA_ENV_SETTINGS
-#undef CONFIG_BOOTCOMMAND
-
-#define CONFIG_EXTRA_ENV_SETTINGS		\
-	"splashpos=m,m\0"			\
-	"fdt_high=0xffffffffffffffff\0"		\
-	"initrd_high=0xffffffffffffffff\0"	\
-
 #endif /* IMX8MQ_EVK_ANDROID_H */
diff --git a/include/configs/imx8mq_evk_androidthings.h b/include/configs/imx8mq_evk_androidthings.h
index cf68349d7b..66d90b5070 100644
--- a/include/configs/imx8mq_evk_androidthings.h
+++ b/include/configs/imx8mq_evk_androidthings.h
@@ -42,12 +42,5 @@
 #define CONFIG_SUPPORT_RAW_INITRD
 #define CONFIG_SERIAL_TAG
 
-#undef CONFIG_EXTRA_ENV_SETTINGS
-#undef CONFIG_BOOTCOMMAND
-
-#define CONFIG_EXTRA_ENV_SETTINGS		\
-	"splashpos=m,m\0"			\
-	"fdt_high=0xffffffffffffffff\0"		\
-	"initrd_high=0xffffffffffffffff\0"	\
 
 #endif /* IMX8MQ_EVK_ANDROID_THINGS_H */
diff --git a/include/configs/imx8qm_arm2_android.h b/include/configs/imx8qm_arm2_android.h
index b2bf785eb0..32c829625c 100644
--- a/include/configs/imx8qm_arm2_android.h
+++ b/include/configs/imx8qm_arm2_android.h
@@ -49,14 +49,6 @@
 #define CONFIG_SUPPORT_RAW_INITRD
 #define CONFIG_SERIAL_TAG
 
-#undef CONFIG_EXTRA_ENV_SETTINGS
-#undef CONFIG_BOOTCOMMAND
-
-#define CONFIG_EXTRA_ENV_SETTINGS					\
-	"splashpos=m,m\0"	  \
-	"fdt_high=0xffffffffffffffff\0"	  \
-	"initrd_high=0xffffffffffffffff\0" \
-
 #define CONFIG_FASTBOOT_BUF_ADDR   CONFIG_SYS_LOAD_ADDR
 #define CONFIG_FASTBOOT_BUF_SIZE   0x19000000
 
diff --git a/include/configs/imx8qxp_arm2_android.h b/include/configs/imx8qxp_arm2_android.h
index cc98c72c6b..abe9d191c7 100644
--- a/include/configs/imx8qxp_arm2_android.h
+++ b/include/configs/imx8qxp_arm2_android.h
@@ -49,14 +49,6 @@
 #define CONFIG_SUPPORT_RAW_INITRD
 #define CONFIG_SERIAL_TAG
 
-#undef CONFIG_EXTRA_ENV_SETTINGS
-#undef CONFIG_BOOTCOMMAND
-
-#define CONFIG_EXTRA_ENV_SETTINGS					\
-	"splashpos=m,m\0"	  \
-	"fdt_high=0xffffffffffffffff\0"	  \
-	"initrd_high=0xffffffffffffffff\0" \
-
 #define CONFIG_FASTBOOT_BUF_ADDR   CONFIG_SYS_LOAD_ADDR
 #define CONFIG_FASTBOOT_BUF_SIZE   0x19000000
 
diff --git a/include/configs/imx8qxp_mek_android.h b/include/configs/imx8qxp_mek_android.h
index 02a22462a4..44c8fcf409 100644
--- a/include/configs/imx8qxp_mek_android.h
+++ b/include/configs/imx8qxp_mek_android.h
@@ -51,13 +51,6 @@
 #define CONFIG_SUPPORT_RAW_INITRD
 #define CONFIG_SERIAL_TAG
 
-#undef CONFIG_EXTRA_ENV_SETTINGS
-#undef CONFIG_BOOTCOMMAND
-
-#define CONFIG_EXTRA_ENV_SETTINGS					\
-	"splashpos=m,m\0"	  \
-	"fdt_high=0xffffffffffffffff\0"	  \
-	"initrd_high=0xffffffffffffffff\0" \
 
 #define CONFIG_FASTBOOT_BUF_ADDR   CONFIG_SYS_LOAD_ADDR
 #define CONFIG_FASTBOOT_BUF_SIZE   0x19000000
-- 
2.17.1

